src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type Props = {
```

Comment:
```suggestion
export type ErrorMessageProps = {
```

Otherwise, we'll get name clashes.